.travis_fedora.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+## Copied script from arpack-ng. Please submit fixes also to
+## arpack-ng (if applicable)
+## -e : Make sure all errors cause the script to fail
+## -x be verbose; write what we are doing, as we do it
+set -ex
+## Should we init a container?
+if [ ".$1" = .setup ] || [ ".$1" = .podman ]
+then
+  # fedora
+  #   note: when you PR, docker-cp provides, in the container, the branch associated with the PR (not master where there's nothing new)
+  #         1. docker create --name mobydick IMAGE CMD        <=> create a container (= instance of image) but container is NOT yet started
+  #         2. docker cp -a ${TRAVIS_BUILD_DIR} mobydick:/tmp <=> copy git repository (CI worker, checkout-ed on PR branch) into the container
+  #                                                               note: docker-cp works only if copy from/to containers (not images)
+  #         3. docker start -a mobydick                       <=> start to run the container (initialized with docker-cp)
+    if test $1 = podman ; then
+	cmd=podman
+    else
+	cmd="sudo docker"
+    fi
+    test . != ".$2" && mpi="$2" || mpi=openmpi
+    test . != ".$3" && version="$3" || version=latest
+    time $cmd pull registry.fedoraproject.org/fedora:$version ||
+	$cmd pull fedora:$version
+    time $cmd create --name mobydick fedora:$version \
+	 /tmp/BOUT-dev/.travis_fedora.sh $mpi
+    time $cmd cp ${TRAVIS_BUILD_DIR} mobydick:/tmp
+    time $cmd start -a mobydick
+    exit 0
+fi
+
+test . != ".$1" && mpi="$1" || mpi=openmpi
+
+## If we are called as root, setup everything
+if [ $UID -eq 0 ]
+then
+    cat /etc/os-release
+    # Ignore weak depencies
+    echo "install_weak_deps=False" >> /etc/dnf/dnf.conf
+    time dnf -y upgrade
+    time dnf -y install dnf-plugins-core {petsc,hdf5}-${mpi}-devel /usr/lib/rpm/redhat/redhat-hardened-cc1
+    time dnf -y builddep bout++
+    useradd test
+    cp -a /tmp/BOUT-dev /home/test/
+    chown -R test /home/test
+    chmod u+rwX /home/test -R
+    restorecon -R /home/test || :
```

Comment:
`/tmp/BOUT-dev/.travis_fedora.sh: line 47: restorecon: command not found` [Travis](https://travis-ci.org/github/boutproject/BOUT-dev/jobs/690147764#L2219)

Is this important? I'm not familiar with `restorecon`

Reply:
No, it is something selinux related. I had initially issues where configure failed. After I noticed `ls /` failed, even as root, i switch to a newer ubuntu, and it worked ...
Have removed it :+1: